pipeline/experiment.py

```diff
@@ -72,7 +72,8 @@ class Photostim(dj.Manual):
     photo_stim :  smallint  # photostim protocol number
     ---
     -> lab.PhotostimDevice
-    power : decimal(4,1)  # mW/mm²
+    power : decimal(4,1)  # Light power at fiber tip (mW) 
+    irradiance=null : decimal(4,1) # mW/mm² (intensity at target location)
```

Comment:
This could be computed in the pipeline, but would require information about the fiber optic cannula (diameter, NA...) and distance to target. Probably better to leave it up to users to computer it beforehand if they want to.